server/routes/serviceProviderReferrals/interventionProgressView.ts

```diff
@@ -1,11 +1,46 @@
-import { TagArgs, SummaryListArgs, SummaryListArgsRow, TableArgs } from '../../utils/govukFrontendTypes'
+import {
+  NotificationBannerArgs,
+  SummaryListArgs,
+  SummaryListArgsRow,
+  TableArgs,
+  TagArgs,
+} from '../../utils/govukFrontendTypes'
 
 import ViewUtils from '../../utils/viewUtils'
 import InterventionProgressPresenter from './interventionProgressPresenter'
+import DateUtils from '../../utils/dateUtils'
 
 export default class InterventionProgressView {
   constructor(private readonly presenter: InterventionProgressPresenter) {}
 
+  private cancelledReferralNotificationBannerArgs(): NotificationBannerArgs {
+    let html = ''
```

Comment:
const vars are preferred everywhere if possible.

Reply:
In this case the fields values are nullable, so therefore I need to default the cancellationReasonHTML and cancellationCommentsHTML to an empty value. This is why I've used `let` here instead of `const`. 

I suppose that if this method is ever called then the endRequestedAt and endRequestedReason should never be null and I could thrown an exception instead.

Reply:
Oh, unless you're just referring to the `html` var, in which case I'll change that to `const`.